src/FacebookPixel/resources/js/frontend/give-fbpt.js

```diff
@@ -1,12 +1,7 @@
-console.log('FBPT script loaded!');
-if (typeof window.fbq !== "undefined") { 
-    console.log('An FB Pixel was detected! The GiveWP Donation Pixel can now fire!');
-    // const form = document.querySelector('form[id*="give-form"]');
-    // const amount = document.querySelector('.give-final-total-amount');
-    // var donateclick = document.getElementById("give-purchase-button");
-    
-    // donateclick.onclick(
-    //     fbq('track', 'Purchase', {currency: form.dataset.currency_code, value: amount}),
-    //     console.log('The GiveWP Pixel fired with a value of:' + form.dataset.currency_code + ' ' + amount)
-    // );
+const { fbq } = window.parent ? window.parent : window;
+const { giveFBPT } = window;
+
+if (typeof fbq !== "undefined") { 
```

Comment:
If you know what something _should_ be, then check for that, as it's a stronger check.
```suggestion
if (typeof fbq === "function") { 
```

Reply:
Great suggestion, done: https://github.com/impress-org/givewp-facebook-pixel-tracking/pull/10/commits/f2331be23254a73d8030212dd34adb2307ae76ab